comment-updater-core/src/main/kotlin/org/jetbrains/research/commentupdater/models/metrics/MetricsCalculator.kt

```diff
@@ -71,27 +67,35 @@ class MetricsCalculator {
             }
         }
 
-        // todo: filter code subtokens from braces, numbers and so on
+        val filterSubTokens = {s: String -> s.any { c -> c.isLetter() }}
```

Comment:
please, make this line more readable. it gets a string and then takes only letters from it? why it is called subtokens, not characters?